src/test/tier_tests.cpp

```diff
@@ -43,24 +43,38 @@ namespace
 
 BOOST_AUTO_TEST_CASE(test_tier_from_outputs_old_tiers)
 {
-    unsigned int tiers[MasternodeTiers::TIER_NONE] = {MasternodeTiers::TIER_1K, MasternodeTiers::TIER_3K, MasternodeTiers::TIER_10K,
-                                                      MasternodeTiers::TIER_30K, MasternodeTiers::TIER_100K, MasternodeTiers::TIER_NONE,
-                                                      MasternodeTiers::TIER_NONE, MasternodeTiers::TIER_NONE, MasternodeTiers::TIER_NONE};
-    bool isMastermodeOutput[MasternodeTiers::TIER_NONE] = {true, true, true, true, true, false, false, false, false};
+    unsigned int tiers[MasternodeTiers::TIER_NONE] = {MasternodeTiers::TIER_NONE, MasternodeTiers::TIER_NONE, MasternodeTiers::TIER_1K, 
```

Comment:
why does it start with 2 TIER_NONE ?
same problem it tests below

Reply:
Fixed